README.md

```diff
@@ -21,12 +21,12 @@ instance methods, and class methods that are supported.
 
 ## features
 
-- Manipulate binary data like a boss, in all browsers -- even IE6!
+- Manipulate binary data like a boss, in all browsers!
 - Super fast. Backed by Typed Arrays (`Uint8Array`/`ArrayBuffer`, not `Object`)
 - Extremely small bundle size (**5.04KB minified + gzipped**, 35.5KB with comments)
-- Excellent browser support (IE 6+, Chrome 4+, Firefox 3+, Safari 5.1+, Opera 11+, iOS, etc.)
+- Excellent browser support (IE, Edge, Chrome, Firefox, Safari, Opera, iOS, etc.)
```

Comment:
Let's list IE11. If we also drop IE9 and IE10 while we're at it, we can kill off the Object implementation of Buffer entirely, and just leave the Uint8Array implementation which would reduce file size significantly.

IE10 technically has Uint8Array but it's buggy, so it uses the Object implementation.

Opera is evergreen, but Safari is not.